packages/matchbox/src/components/Spinner/Spinner.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { margin, layout, compose } from 'styled-system';
+import { pick } from '@styled-system/props';
+import { createPropTypes } from '@styled-system/prop-types';
+import { Box } from '../Box';
+import { ScreenReaderOnly } from '../ScreenReaderOnly';
+import { circleOuter, circle } from './styles';
+
+const system = compose(margin, layout);
```

Comment:
Since `size` is also a system prop I see this in the css - I think all we need is height and width right? Do you think we still need this? I might have wrote the AC wrong in the ticket, I _think_ I meant `position` not `layout` 🤦 


<img width="146" alt="Screen Shot 2020-05-26 at 6 56 08 PM" src="https://user-images.githubusercontent.com/3903325/82958121-ae79bc80-9f82-11ea-97be-1643450cc5b7.png">

Reply:
Yea that makes more sense. Since we've got size defined in a prop we won't need to change that with the `layout` prop.